<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAjouterVente</class>
 <widget class="QDialog" name="DialogAjouterVente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>689</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>689</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>199</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ajouter un produit en vente</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNom">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>122</y>
     <width>201</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="valNom">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>122</y>
     <width>146</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelCat">
   <property name="geometry">
    <rect>
     <x>116</x>
     <y>156</y>
     <width>201</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Catégorie :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="valCat">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>155</y>
     <width>146</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelPrix">
   <property name="geometry">
    <rect>
     <x>116</x>
     <y>196</y>
     <width>201</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Prix Unitaire/Prix de départ :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelQte">
   <property name="geometry">
    <rect>
     <x>116</x>
     <y>226</y>
     <width>201</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Quantité :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="valQte">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>221</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>237</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelDateLimite">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>116</x>
     <y>266</y>
     <width>201</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Date limite si enchère :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioEnchere">
   <property name="geometry">
    <rect>
     <x>375</x>
     <y>70</y>
     <width>164</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Vente aux enchères</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QDateEdit" name="valDateLimite">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>260</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2015</year>
     <month>12</month>
     <day>25</day>
    </datetime>
   </property>
   <property name="minimumDateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2015</year>
     <month>12</month>
     <day>1</day>
    </datetime>
   </property>
  </widget>
  <widget class="QPushButton" name="boutonAjouterVente">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>350</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Mettre en vente</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioNormale">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>164</width>
     <height>22</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Vente normale</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QDoubleSpinBox" name="valPrix">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>190</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <double>1.000000000000000</double>
   </property>
   <property name="maximum">
    <double>10000.000000000000000</double>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>radioEnchere</sender>
   <signal>toggled(bool)</signal>
   <receiver>labelDateLimite</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>456</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioEnchere</sender>
   <signal>toggled(bool)</signal>
   <receiver>valDateLimite</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>456</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>375</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
